products/nixos.md

```diff
@@ -0,0 +1,91 @@
+---
+title: NixOS
+layout: post
+category: os
+iconSlug: nixos
+permalink: /nixos
+alternate_urls:
+  - /nixoslinux
+  - /nix
```

Comment:
Maybe we should keep this one free for Nix itself?

Reply:
Yeah good point